ignite/handlers/param_scheduler.py

```diff
@@ -465,12 +465,28 @@ def print_lr():
 
     .. versionchanged:: 0.4.13
         Added cyclic warm-up to the scheduler using ``warmup_duration``.
+
+    .. versionchanged:: 0.5.0
+        Added monotonic argument ....
```

Comment:
Here instead of `....` we need to write a proper sentence

Reply:
Actually, I was curious about meaning of `....`. I am gonna add proper docstring :)